ansys/mapdl/core/launcher.py

```diff
@@ -911,6 +914,19 @@ def launch_mapdl(exec_file=None, run_location=None, jobname='file',
         start_parm['override'] = override
         start_parm['timeout'] = start_timeout
 
+    # Here we will check the license server
+    if license_server_check:
+
+        if os.getenv('ANSYSLMD_LICENSE_FILE') is not None:
+            # We are in a dockerized environment. 
+            # We skip processing for the moment.
+            pass 
+
+        else:
+            # Running locally 
+            license_thread = threading.Thread(target=try_license_server)
+            license_thread.start()
```

Comment:
```suggestion
    if license_server_check and os.getenv("ANSYSLMD_LICENSE_FILE") is None:
            # Running locally 
            license_thread = threading.Thread(target=try_license_server)
            license_thread.start()
```
This should be the same logic and is simplified.

Reply:
Good suggestion.